.travis.yml

```diff
@@ -26,22 +26,21 @@ before_install:
         wget https://s3.amazonaws.com/octave-snapshot/public/octave-ubuntu-trusty-snapshot.tar.xz;
         sudo tar --extract --directory=/usr/local --strip-components=1 --file=octave-ubuntu-trusty-snapshot.tar.xz;
     fi
-  - sudo apt-get install -y python$PYTHON_VER-pip
+  - sudo apt-get install -y python$PYTHON_VERSION-pip
   - if [ "x$PYTAVE" = "xyes" ]; then
-        sudo apt-get install -y libboost-python-dev python$PYTHON_VER-numpy;
+        sudo apt-get install -y python$PYTHON_VERSION-dev python$PYTHON_VERSION-numpy;
     fi
 
 install:
-  - pip$PYTHON_VER install --user sympy==$SYMPY_VER
+  - pip$PYTHON_VERSION install --user sympy==$SYMPY_VER
   - if [ "x$DOCTEST" = "xyes" ]; then
         mkdir ${HOME}/octave;
         wget https://github.com/catch22/octave-doctest/releases/download/v0.5.0/doctest-0.5.0.tar.gz;
         octave -W --no-gui --eval "pkg install doctest-0.5.0.tar.gz";
     fi
   - if [ "x$PYTAVE" = "xyes" ]; then
-        hg clone https://bitbucket.org/genuinelucifer/pytave_main pytave;
+        hg clone https://bitbucket.org/mtmiller/pytave pytave;
```

Comment:
The build can be done with the main repo. No need for a proxy repo and bookmark.